roles/postgresql/tasks/main.yml

```diff
@@ -145,26 +199,55 @@
   when:
     - postgres_host is not none
     - postgresql_is_local
+    - ansible_os_family == "Debian"
     - running_on_server
   become: true
   notify:
     - reload local postgres
 
-- include_tasks: pg_hba.yml
-- include_tasks: warm_standby_leader.yml
+- name: PostgreSQL | ensure access to postgres server (RedHat)
+  ansible.builtin.blockinfile:
+    path: '/var/lib/pgsql/data/pg_hba.conf'
+    insertafter: "# Put your actual configuration here"
+    block: |
+      # localhost
+  when:
+    - postgres_host is not none
+    - postgresql_is_local
+    - ansible_os_family == "RedHat"
+    - running_on_server
+  become: true
+  notify:
+    - restart local postgres
+
+- name: PostgreSQL | configure access to postgres server
+  ansible.builtin.include_tasks: pg_hba.yml
+
+- name: PostgreSQL | configure warm standy
+  ansible.builtin.include_tasks: warm_standby_leader.yml
   when:
     - postgresql_cluster != false
     - inventory_hostname == postgresql_cluster.leader
-- include_tasks: warm_standby_follower.yml
+
+- name: PostgreSQL | configure warm standy follower
+  ansible.builtin.include_tasks: warm_standby_follower.yml
   when:
     - postgresql_cluster != false
     - inventory_hostname in postgresql_cluster.followers
-- include_tasks: logical_replication.yml
+
+- name: PostgreSQL | configure logical replication
+  ansible.builtin.include_tasks: logical_replication.yml
   when:
     - postgresql_cluster != false
     - inventory_hostname == postgresql_cluster.leader
     - postgresql_logical_replication != false
-- include_tasks: create_db.yml
-- include_tasks: create_users.yml
-- include_tasks: create_read_only_user.yml
```

Comment:
By moving these out of this mega-task, are we losing the conditions? With this change, I think the `create_db` and `create_users` includes would run on all boxes. Maybe a block is the right answer here?